src/main/scala/intellij/haskell/editor/HaskellCompletionContributor.scala

```diff
@@ -392,6 +402,43 @@ class HaskellCompletionContributor extends CompletionContributor {
     waitForResult(lookupElements)
   }
 
+  private def getIdsFromAllExportedModules(project: Project) = {
```

Comment:
But how do you know context? I think the only useful solution is to make action to load current file in Console repl.